src/packages/@ncigdc/modern_components/GeneExpression/GeneExpressionChart.js

```diff
@@ -7,17 +7,34 @@ import { theme } from '@ncigdc/theme';
 import './inchlib';
 
 const category_colors = {
```

Comment:
Wondering whether these should be abstracted to a centralised constants file elsewhere, accessible globally so we can reuse them.
Thoughts?

Reply:
that's a good idea, will do on PRTL-2962 (will be merged in here)